ckan/config/middleware/flask_app.py

```diff
@@ -69,6 +69,24 @@
     )
 
 
+class CKANSession(Session):
+    def _get_interface(self, app: CKANApp):
+        """Initialize session interface.
+
+        If config file contains custom import path for ``SESSION_TYPE``, use it
+        to allow session customizations. If custom import path is missing,
+        initialize one of interfaces provided by flask-session module.
+
+        """
+        session_type = app.config["SESSION_TYPE"]
+        option_name = f"ckan.session.custom_type.{session_type}.import_path"
+
+        if path := config.get(option_name):
+            return import_string(path)(app)
```

Comment:
Importing arbitrary python code from the INI/environment variables doesn't align with ckan's current pattern of implementing interfaces and enabling plugins. Is there a reason for this different approach here?

Reply:
If there isn't an immediate need for this interface I suggest we drop it and discuss the best way to implement pluggable session types. `import_string(s)` is roughly the same as `exec(s)` so I'm leery about including it here.

Reply:
Yes, perhaps adding support for custom interfaces is premature. The whole point of Flask-Session is that provides many options already out of the box.
I was also going to suggest that we remove the [`ckan.session.custom_type.cookie.import_path`](https://github.com/ckan/ckan/pull/7893/files#diff-5752be3d55839c292941807dff4cb1943222e276a2d2a64888e23f5b49104628R343) and [`ckan.session.custom_type.redis.import_path`](https://github.com/ckan/ckan/pull/7893/files#diff-5752be3d55839c292941807dff4cb1943222e276a2d2a64888e23f5b49104628R351) options. The wrappers @smotornyuk introduced make perfect sense in the context of CKAN, I don't think they need to be customizable.